Fail to install on MAC OS node 18  / 16 / 14

```
npm install node-magickwand --build-from-source

npm ERR! code 1
npm ERR! path /Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand
npm ERR! command failed
npm ERR! command sh -c node-pre-gyp install --fallback-to-build && node scripts/gen-es6.js lib/index.mjs
npm ERR! Failed to execute '/Users/macuser/.nvm/versions/node/v18.14.0/bin/node /Users/macuser/.nvm/versions/node/v18.14.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand/lib/binding/darwin-x64/node-magickwand.node --module_name=node-magickwand --module_path=/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand/lib/binding/darwin-x64 --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v108' (1)
npm ERR! node-pre-gyp info it worked if it ends with ok
npm ERR! node-pre-gyp info using node-pre-gyp@1.0.11
npm ERR! node-pre-gyp info using node@18.14.0 | darwin | x64
npm ERR! node-pre-gyp info build requesting source compile
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.0
npm ERR! gyp info using node@18.14.0 | darwin | x64
npm ERR! gyp info ok 
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.0
npm ERR! gyp info using node@18.14.0 | darwin | x64
npm ERR! gyp info find Python using Python version 3.8.2 found at "/Applications/Xcode.app/Contents/Developer/usr/bin/python3"
npm ERR! gyp info spawn /Applications/Xcode.app/Contents/Developer/usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/Users/macuser/.nvm/versions/node/v18.14.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/macuser/.nvm/versions/node/v18.14.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/macuser/Library/Caches/node-gyp/18.14.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/Users/macuser/Library/Caches/node-gyp/18.14.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/Users/macuser/.nvm/versions/node/v18.14.0/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/Users/macuser/Library/Caches/node-gyp/18.14.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/Library/Python/3.8/site-packages/pygments'
npm ERR! Consider using the `--user` option or check the permissions.
npm ERR! 
npm ERR! WARNING: You are using pip version 19.2.3, however version 23.2.1 is available.
npm ERR! You should consider upgrading via the 'pip install --upgrade pip' command.
npm ERR! gyp: Call to '(pip3 install "conan<2.0.0" && cd build && conan install .. -pr:b=default -of build --build=missing --build=openjpeg) > /dev/null' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
npm ERR! gyp ERR! configure error 
npm ERR! gyp ERR! stack Error: `gyp` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onCpExit (/Users/macuser/.nvm/versions/node/v18.14.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:325:16)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
npm ERR! gyp ERR! System Darwin 19.6.0
npm ERR! gyp ERR! command "/Users/macuser/.nvm/versions/node/v18.14.0/bin/node" "/Users/macuser/.nvm/versions/node/v18.14.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand/lib/binding/darwin-x64/node-magickwand.node" "--module_name=node-magickwand" "--module_path=/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand/lib/binding/darwin-x64" "--napi_version=8" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v108"
npm ERR! gyp ERR! cwd /Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand
npm ERR! gyp ERR! node -v v18.14.0
npm ERR! gyp ERR! node-gyp -v v9.3.0
npm ERR! gyp ERR! not ok 
npm ERR! node-pre-gyp ERR! build error 
npm ERR! node-pre-gyp ERR! stack Error: Failed to execute '/Users/macuser/.nvm/versions/node/v18.14.0/bin/node /Users/macuser/.nvm/versions/node/v18.14.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand/lib/binding/darwin-x64/node-magickwand.node --module_name=node-magickwand --module_path=/Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand/lib/binding/darwin-x64 --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v108' (1)
npm ERR! node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/macuser/practices/imageCompression/magick/node_modules/@mapbox/node-pre-gyp/lib/util/compile.js:89:23)
npm ERR! node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
npm ERR! node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1091:16)
npm ERR! node-pre-gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:302:5)
npm ERR! node-pre-gyp ERR! System Darwin 19.6.0
npm ERR! node-pre-gyp ERR! command "/Users/macuser/.nvm/versions/node/v18.14.0/bin/node" "/Users/macuser/practices/imageCompression/magick/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
npm ERR! node-pre-gyp ERR! cwd /Users/macuser/practices/imageCompression/magick/node_modules/node-magickwand
npm ERR! node-pre-gyp ERR! node -v v18.14.0
npm ERR! node-pre-gyp ERR! node-pre-gyp -v v1.0.11
npm ERR! node-pre-gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/macuser/.npm/_logs/2023-10-05T19_36_22_455Z-debug-0.log
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+### [0.9.7]
+
+ - Fix [#39](https://github.com/mmomtchev/node-magickwand/issues/39), do not install `conan` in the system directory
+
 ### [0.9.6] 2023-10-04
 
  - Update ImageMagick to 7.1.1-19

--- a/README.md
+++ b/README.md
@@ -139,7 +139,11 @@ The `Image.display()` function works and it is an excellent debugging tool. On m
 npm install node-magickwand --build-from-source
 ```
 
-You will need a working C++11 environment. On Windows nothing but VS 2022 works at the moment. This will also rebuild the included Magick++ library.
+You will need a working C++11 environment.
+
+On Windows nothing but VS 2022 works at the moment. This will also rebuild the included Magick++ library.
+
+On Linux and macOS it uses `pip3` to install the `conan` module which builds a number of required libraries in `${HOME}/.conan`. After building, you can safely delete this directory if you wish, since `node-magickwand` is statically linked.
 
 ### Rebuilding from git or using an externally provided ImageMagick library
 

--- a/binding.gyp
+++ b/binding.gyp
@@ -177,7 +177,7 @@
             # This an ugly hack that enable running of shell commands during node-gyp configure
             # node-gyp configure needs to evaluate this expression to generate the platform-specific files
             # (originally by TooTallNate for libffi) 
-            '<!@((pip3 install --user "conan<2.0.0" && cd build && conan install .. -pr:b=default -of build --build=missing --build=openjpeg) > /dev/null)',
+            '<!@((pip3 install --user "conan<2.0.0" && cd build && python3 -m conans.conan install .. -pr:b=default -of build --build=missing --build=openjpeg) > /dev/null)',
             '<!@(bash configure_magick.sh <(module_path) <(hdri))'
           ]
         }

--- a/binding.gyp
+++ b/binding.gyp
@@ -177,7 +177,7 @@
             # This an ugly hack that enable running of shell commands during node-gyp configure
             # node-gyp configure needs to evaluate this expression to generate the platform-specific files
             # (originally by TooTallNate for libffi) 
-            '<!@((pip3 install "conan<2.0.0" && cd build && conan install .. -pr:b=default -of build --build=missing --build=openjpeg) > /dev/null)',
+            '<!@((pip3 install --user "conan<2.0.0" && cd build && conan install .. -pr:b=default -of build --build=missing --build=openjpeg) > /dev/null)',
             '<!@(bash configure_magick.sh <(module_path) <(hdri))'
           ]
         }

